### PR TITLE
Created desktop version

### PR DESCRIPTION
**Pull request by 10-menachi**
Added Styled version of the: 
- Main Section
- Work Section
- About me Section
- Contact Me Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,10 +29,6 @@
           <li><a href="#contact">Contact</a></li>
         </ul>
       </div>
-
-      <nav class="ham-menu">
-        <p class="menu-icon">&#9776;</p>
-      </nav>
```
Comment (algerina) [inline index.html]: - [ ] Well done working on the desktop version of your Portfolio however the mobile version should work properly as well. As you can see on the caption the hamburger menu on the mobile version is not displayed, kindly work on that issue.

port1

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @10-menachi ,

Great job so far! 
However, you still have minor issues to fix.

May I highlight the following: 
- [x] Your project looks awesome👏  ✔️ 
- [x] No Linter errors✔️ 
- [x] Well done implementing most of the previous reviewer's requirements✔️ 



## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by joel-kalema — APPROVED** [review]
Hi @10-menachi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bifabig — COMMENTED** [review]
Hi @10-menachi 

Great Job 👍 

You have designed the portfolio landing page to perfection, it is well-responsive!!

**Review by katfogy — COMMENTED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @10-menachi  ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 

- No linter errors :+1: 
- Good use of GitHub flow :+1: 
-  Good job generating a Formspree id :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 10-menachi** [comment]
Big thanks to all of you :heart:

**Pull request merged**